CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Unreleased
 * Upgraded mini extract text plugin
 * Ignore css order warnings from mini extract text plugin
 * Upgraded terra-node to node 10
-* Improve afterCommand error handling
```

Comment:
Was this removed accidentally?

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/317/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e
Looks like it got lost is merge with master